MapView/Map/RMConfiguration.h

```diff
@@ -47,6 +47,9 @@
 /** A Mapbox API access token. Obtain an access token on your [Mapbox account page](https://www.mapbox.com/account/apps/). */
 @property (nonatomic, retain) NSString *accessToken;
 
+/** A default URL session for the framework */
+@property (nonatomic, readonly) NSURLSession *mapBoxUrlSession;
```

Comment:
It's an artefact from times where I had here two sessions: the general one and the backgroundSession. Now it's without any sense, you're right. Changed in 153d737.